### PR TITLE
db/fixed-lang-options

### PR DESCRIPTION
**Pull request by sebastianbochan**

### CODE REVIEW

**Review by pawellysy — APPROVED** [review]
Looks Good!

**Review by karolkolodziej — APPROVED** [review]
Looks good but I have the same questions as Pawel.
Is the demo going to be a separate PR from someone else?

**Review by bre1470 — APPROVED** [review]

File: samples/dashboards/cypress/options-lang/demo.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!-- <script src="https://code.highcharts.com/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
```
Comment (karolkolodziej) [inline samples/dashboards/cypress/options-lang/demo.html:2]: Remove?

**Review by karolkolodziej — APPROVED** [review]
Shouldn't we also have a demo with that for the distribution? I mean not only the `.mjs`

File: samples/dashboards/cypress/options-lang/demo.html (hunk 1 above)
Comment (sebastianbochan) [reply samples/dashboards/cypress/options-lang/demo.html:2]: Done, but it was not a part of this task ;)

**Comment by sebastianbochan** [comment]
> Shouldn't we also have a demo with that for the distribution? I mean not only the `.mjs`

We can create a separate ticket for that.

**Pull request merged**